lit/setup-and-operations/teams.lit

```diff
@@ -6,13 +6,12 @@ Every pipeline and one-off build belongs to a \italic{team}.
 
 A team is simply a conceptual owner and a separate namespace for data in
 Concourse (pipelines, builds, etc.) with its own auth configuration. A team may
-map to GitHub oAuth, basic auth, UAA auth, or whatever other auth methods we
-add in the future.
+whitelist users or groups either from local auth or from an external provider
```

Comment:
I'd avoid the term "groups" here (since we're abstracting that aspect away with our own CLI) - how about:

A team is simply a conceptual owner and a separate namespace for data in Concourse (pipelines, builds, etc.). Individual user access to a team is determined by how it's configured with `set-team` - either by whitelisting local users or configuring external providers that are enabled for the Concourse server (see \reference{auth-config}).